2020/day-3-toboggan-trajectory/index.ts

```diff
@@ -0,0 +1,23 @@
+type Movements = [right: number, down: number]
```

Comment:
Interesting, what is the `[right: number, down: number]` doing here?

Reply:
[Labeled Tuple Elements](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-0.html#labeled-tuple-elements). An array type with a fixed number of elements whose types are known is called a [tuple](https://www.typescriptlang.org/docs/handbook/basic-types.html#tuple), which in this case would look like this: `[number, number]`. A recent addition to TS allows you to add labels to these which is what I've done here.